spec/requests/whitehall_migration_spec.rb

```diff
@@ -0,0 +1,46 @@
+RSpec.describe "Whitehall Migration" do
+  let(:whitehall_migration) { create(:whitehall_migration) }
+  let(:document) { create(:whitehall_migration_document_import, whitehall_migration: whitehall_migration) }
```

Comment:
It'd be good to call this a document_import rather than a document since document is already a word with quite a lot of meaning in Content Publisher.